libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1409,23 +1417,27 @@ string YulUtilFunctions::storageArrayPushFunction(ArrayType const& _type)
 					default {
 						sstore(array, add(data, 2))
 						let slot, offset := <indexAccess>(array, oldLen)
-						<storeValue>(slot, offset, value)
+						<storeValue>(slot, offset, <values>)
 					}
 				<!isByteArray>
 					let oldLen := sload(array)
 					if iszero(lt(oldLen, <maxArrayLength>)) { <panic>() }
 					sstore(array, add(oldLen, 1))
 					let slot, offset := <indexAccess>(array, oldLen)
-					<storeValue>(slot, offset, value)
+					<storeValue>(slot, offset, <values>)
 				</isByteArray>
 			})")
 			("functionName", functionName)
+			("values", suffixedVariableNameList("value", 0, _fromType->sizeOnStack()))
```

Comment:
What if `sizeOnStack` is 1 - can this generate the string `value`?

Reply:
It generates `value_0` if stack size is 1

Reply:
But the list can be empty, so I think we need to be prepared for there to be no comma. I'll change that.